client/homebrew/pages/basePages/listPage/brewItem/brewItem.jsx

```diff
@@ -90,11 +91,15 @@ const BrewItem = createClass({
 		</a>;
 	},
 
-	renderGoogleDriveIcon : function(){
-		if(!this.props.brew.googleId) return;
+	renderStorageIcon : function(){
+		if(!this.props.brew.googleId) return <span title='Internal Homebrewery Database'>
```

Comment:
Change the title to just `Homebrewery Storage`.

Reply:
What about NaturalCrit Storage?

Reply:
Either way. But I think elsewhere (the Account page) we refer to it as Homebrewery Storage already.